doc/manual/syntax.md

```diff
@@ -0,0 +1,313 @@
+| +        | The addition operator                                | `1 + 2 = 3`   |
+| \-       | The subtraction operator                             | `1 - 2 = -1`  |
+| *        | The multiplication operator                          | `1 * 2 = 2`   |
+| /        | The division operator                                | `1 / 2 = 0.5` |
+| %        | The modulo operator (returns the *signed* remainder) | `5 % 3 = 2`   |
+
+> **Remark about the `-` operator:***  
+> Since `-` can be used inside of an identifier, the subtraction operators **needs** to be surrounded by spaces,
+> write `1 - 1`, not `1-1`.
+
+Numbers are also comparable, using the dedicated comparison operators, returning a boolean value :
+| Operator | Description      | Example   |
+|:--------:|:----------------:|:---------:|
+| ==       | Equal            | `5 == 5`  |
+| !=       | Not Equal        | `5 != 4`  |
+| <        | Smaller than     | `2 < 3`   |
+| >        | Greater than     | `1 > -5`  |
+| >=       | Greater or Equal | `1 >= 1`  |
+| <=       | Smaller or Equal | `-1 <= 6` |
+
+All those operators have precedence with each others. In the table below, you will find the operators sorted from highest to lowest precedence:
+|       Operators      | Associativity | Remark                                        |
+|:--------------------:|:-------------:|-----------------------------------------------|
+|       `( ... )`      |               | parentheses always have to highest precedence |
+|          `-`         | right-to-left | unary negation (as in `-1`)                   |
+|     `*`, `/`, `%`    | left-to-right |                                               |
+|       `+`, `-`       | left-to-right | binary addition and subtraction               |
+| `<`, `>`, `=<`, `>=` | left-to-right |                                               |
+|      `==`, `!=`      | left-to-right |                                               |
+
+### Boolean values
+
+The boolean values in Nickel are denoted `true` and `false`.
+
+To work with boolean values, we have the classic logical operators *AND* (&&), *OR* (||) and *NOT* (!).
+The _AND_ and _OR_ operators are lazy in the evaluation of the second argument. The truth table below gives the results for using these operators:
+
+Examples:
+```
+> true && false
+false
+
+> false || true
+true
+
+> ! true
+false
+```
+
+### Strings
+
+Nickel can work with sequences of characters, or strings. 
+Strings are enclosed by `" ... "` for a single line string or by `m#" ... "#m` for a multiline string.
+They can be concatenated with the operator `++`.
+Strings must be UTF-8 valid.
+
+String interpolation exists in Nickel, via the notation `"#{ < expression that evaluates to a string > }"`.
+
+Examples:
+```
+> "Hello, World!"
+"Hello, World!"
+
+> m#"Well, if this isn't a multiline string?
+Yes it is, indeed it is"#m
+"Well, if this isn't a string?
+Yes it is, indeed it is"
```

Comment:
It could be nice to have multiline strings inside indented code, to see the difference:
```
{
  program = {
    shell_code = m#"
      if [ -f /var/log/messages ]; then
          echo "/var/log/messages exists."
      fi
    "#m
  }
}
```
then `program.shell_code` is:
```
"if [ -f /var/log/messages ]; then
  echo "/var/log/messages exists."
fi"

Reply:
added an example